src/main/resources/schema.sql

```diff
@@ -44,6 +44,8 @@ CREATE TABLE IF NOT EXISTS interview
     created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
     updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
 );
+ALTER TABLE interview ADD COLUMN IF NOT EXISTS timer INT;
```

Comment:
```suggestion
ALTER TABLE interview ADD COLUMN IF NOT EXISTS timer INT NOT NULL;
```
NOT NULL 조건 붙여주는게 어떨까요~?